class.jetpack-onboarding-end-points.php

```diff
@@ -539,6 +545,41 @@ static function install_theme() {
 		wp_send_json_success( $theme_id );
 	}
 
+	static function install_plugin( $slug ) {
+		if ( is_multisite() && ! current_user_can( 'manage_network' ) ) {
```

Comment:
In addition to these, we should add `|| ! current_user_can( 'install_plugins' )` - we want to avoid allowing users without the necessary capabilities to install a plugin.